contract/tests/test_portal.py

```diff
@@ -16,10 +16,22 @@ def test_tour(self):
         contract = self.env["contract.contract"].create(
             {"name": "Test Contract", "partner_id": partner.id}
         )
-        user_portal = self.env.ref("base.demo_user0")
+        portal_group = self.env.ref("base.group_portal")
+        user_portal = (
```

Comment:
There's a method in `BaseCommon` for creating easily a user. Please use it instead.